### PR TITLE
Enhancement: Track Link Tap in Auction Results For You

### PR DESCRIPTION
**Pull request by lordkiz**
The type of this PR is: **TYPE**

This PR resolves [CX-1698]

### Description
- Track link tap and Auction Results Screen

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Track link tap and AuctionResultsForYou Screen - kizito

</details>

[CX-1698]: https://artsyproduct.atlassian.net/browse/CX-1698

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Track link tap and AuctionResultsForYou Screen - kizito)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 15940800df1b724749ec0230b79596615a4c1010
</p>

File: src/lib/Scenes/AuctionResultsForYou/AuctionResultsForYou.tsx [hunk 1]
```diff
@@ -190,4 +199,10 @@ export const tracks = {
     destination_screen_owner_id: auctionResultId,
     type: "thumbnail",
   }),
+  tappedLink: tappedLink({
+    contextModule: ContextModule.auctionResultsForArtistsYouFollow,
+    contextScreenOwnerType: OwnerType.auctionResultsForArtistsYouFollow,
+    destinationScreenOwnerSlug: "/favorites",
```
Comment (olerichter00) [inline src/lib/Scenes/AuctionResultsForYou/AuctionResultsForYou.tsx:205]: I don't think we need the slug here. Usually, we send the slug of an artwork or an artist, but in this case, it's an overview page without a specifice slug.

**Review by olerichter00 — APPROVED** [review]
Looks good! I just don't think we need to send the slug (see comment).

File: src/lib/Scenes/AuctionResultsForYou/AuctionResultsForYou.tsx (hunk 1 above)
Comment (lordkiz) [reply src/lib/Scenes/AuctionResultsForYou/AuctionResultsForYou.tsx:205]: Thanks @olerichter00 . Removing...

**Pull request merged**